commands/push-image.ts

```diff
@@ -26,6 +31,19 @@ export async function pushImage(context?: ImageNode): Promise<void> {
         }
     }
 
+    if (imageName.includes('/')) {
+        prefix = imageName.substring(0, imageName.lastIndexOf('/'));
+    }
+    if (prefix && askToSavePrefix !== false) { //account for undefined
+        let userPrefixPreference: vscode.MessageItem = await ext.ui.showWarningMessage("Would you like to save the prefix for autocomplete later?", DialogResponses.yes, DialogResponses.no, DialogResponses.skipForNow);
+        if (userPrefixPreference === DialogResponses.yes || userPrefixPreference === DialogResponses.no) {
+            askToSavePrefix = false;
+        }
+        if (userPrefixPreference === DialogResponses.yes) {
+            await configOptions.update('defaultRegistryPath', prefix, vscode.ConfigurationTarget.Workspace);
+        }
+        await configOptions.update('askToSavePrefix', askToSavePrefix, vscode.ConfigurationTarget.Workspace);
```

Comment:
nit: I'd just do 

await configOptions.update('askToSavePrefix', false, vscode.ConfigurationTarget.Workspace); 

instead of 'askToSavePrefix = false' above, rather than updating it even when you're not actually changing its value.

Reply:
And it's also high priority to avoid writing when not needed now that we know updating a setting could write it into a different place than it currently is.